src/gleam_stdlib.erl

```diff
@@ -498,6 +498,9 @@ inspect_maybe_utf8_string(Binary, Acc) ->
                 $\n -> <<$\\, $n>>;
                 $\t -> <<$\\, $t>>;
                 $\f -> <<$\\, $f>>;
+                $\b -> <<$\\, $b>>;
+                $\v -> <<$\\, $v>>;
+                $\e -> <<$\\, $e>>;
```

Comment:
These are not valid Gleam syntax. They'll need to be written using the `\u{}` syntax

Reply:
Sorry, I've been unclear.

`string.inspect` is supposed to render Gleam syntax, not Erlang syntax, so we need to use the Gleam escape syntax for these byte sequences. The escaped string should be valid Gleam syntax.